### PR TITLE
Raz/mgmt 2038 enable disable transition for update

### PR DESCRIPTION
**Pull request by RazRegev**

### CODE REVIEW

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -1859,7 +1863,7 @@ func (b *bareMetalInventory) refreshClusterStatus(
 	var c common.Cluster
 
 	filter := identity.AddUserFilter(ctx, "id = ?")
-	if err := db.First(&c, filter, clusterID).Error; err != nil {
+	if err := db.Preload("Hosts").First(&c, filter, clusterID).Error; err != nil {
```
Comment (ori-amizur) [inline internal/bminventory/inventory.go:1866]: Why is this change needed?

**Review by YuviGold — APPROVED** [review]

**Review by filanov — APPROVED** [review]

**Review by YuviGold — APPROVED** [review]

**Comment by RazRegev** [comment]
job is hanged, will reopen
